package.json

```diff
@@ -35,13 +35,13 @@
     "babel-plugin-istanbul": "^6.0.0",
     "mocha": "^6.2.3",
     "mocha-steps": "^1.3.0",
+    "node": "^13.14.0",
```

Comment:
Moved node to dev dependencies. It was causing eb deployments to fail because node was already installed.